### ISSUE TITLE
Cannot insert boolean type value into table

### ISSUE THREAD

**Issue by malin1993ml** [cmu-db/peloton#381]
```sql
postgres=# create table test(a int, b boolean);
create table test(a int, b boolean) 0
postgres=# insert into test values(0, true);
Error parsing SQL statement
postgres=# insert into test values(0, 'true');
Can't serialize boolean types to storage
```

**Comment by elvismt** [comment]
I was looking for something to do in order to contribute to peloton and when I tried to reproduce this bug I figured out that this is not happening anymore. Probably due to your code improvements since Dec 2016. PSQL section reproduced below. I would appreciate if anybody suggest me a lower complexity task to get started.

```sql
elvis=# create table foo(a int, b boolean);
create table foo(a int, b boolean) 0
elvis=# insert into foo values(1, 'true');
insert into foo values(1, 'true') 1
elvis=# select * from foo;
 a |  b   
---+------
 1 | true
(1 registro)

elvis=# insert into foo values(2, 'false');
insert into foo values(2, 'false') 1
elvis=# select * from foo;
 a |   b   
---+-------
 1 | true
 2 | false
(2 registros)
```

**Comment by apavlo** [comment]
I must have fixed this in 67b6a1a78ac51e38d99674fcfdf6dc5de47115ff

Good to know.

I will send you a separate email later this week.
